packages/react-examples/src/react/ContextualMenu/ContextualMenu.CustomMenuList.Example.tsx

```diff
@@ -83,7 +84,17 @@ export const ContextualMenuWithCustomMenuListExample: React.FunctionComponent =
     [items, renderMenuList, onDismiss],
   );
 
-  return <DefaultButton text="Click for ContextualMenu" menuProps={menuProps} />;
+  return (
+    <>
+      <p>
+        <Text>
+          Warning: adding an input to a menu is not technically allowed, and will trigger errors in Accessibility
+          Insights and axe-core. Accessibility issues in practice are minimal, however.
```

Comment:
Nit: Consider rephrasing in active voice:

"However, real world accessibility issues are minimal and or non-issues."

Reply:
done!